lib/pacemaker/pcmk_resource.c

```diff
@@ -228,6 +228,7 @@ pcmk__resource_digests(pcmk__output_t *out, pcmk_resource_t *rsc,
     return rc;
 }
 
+// @COMPAT The scheduler parameter is unused and can be removed at the next break
```

Comment:
Yeah we should have designed this one better all around. As a high-level function, it should probably take the resource and node names instead of objects, and connect to the CIB to get the objects. Unrelated to this PR though.